main.sh

```diff
@@ -32,8 +32,8 @@ if [ "${BASH_SOURCE[0]}" != "${0}" ]; then
     __b3bp_tmp_source_idx=1
   fi
 else
-  [ ! -z "${__usage+x}" ] && unset __usage
-  [ ! -z "${__helptext+x}" ] && unset __helptext
+  [ ! -z "${__usage+x}" ] && unset -v __usage
+  [ ! -z "${__helptext+x}" ] && unset -v __helptext
```

Comment:
Is +x here just to handle the case where __usage or __helptext never get set, therefore preventing them from being unset without being set in the first place?